cmd/pebble/cmd_mkdir.go

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2022 Canonical Ltd
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 3 as
+// published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/jessevdk/go-flags"
+
+	"github.com/canonical/pebble/client"
+)
+
+type cmdMkdir struct {
+	clientMixin
+
+	MakeParents bool   `short:"p" long:"parents"`
+	Permissions string `short:"m" long:"mode"`
+	User        string `short:"u" long:"user"`
+	Group       string `short:"g" long:"group"`
```

Comment:
With the `exec` command we discussed this, and ended up going for two params, `--uid` and `--user` (and `--gid` and `--group`). Let's do the same here for consistency, and drop the short `u` and `g` variants.